### PR TITLE
Placeholders for pickers

### PR DESCRIPTION
**Pull request by alrod**

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "contentVersion": "2016-03-04-alpha",
     "variables": {
         "storageConnStringLabel": "Storage account connection",
-        "appSettingsHelp": "This is the connection string for your storage account."
+        "appSettingsHelp": "This is the connection string for your storage account.",
+        "selectConnectoin":  "Click select to choose a connection"
```
Comment (soninaren) [inline Bindings/bindings.json]: typing error at "selectConnectoin"

File: Bindings/bindings.json [hunk 2]
```diff
@@ -50,7 +51,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Event Hub connection",
-                    "help": "This is the connection string for your Event Hub. Make sure that it has Receive permissions."
+                    "help": "This is the connection string for your Event Hub. Make sure that it has Receive permissions.",
+                    "placeholder": "[variables('selectConnectoin')]"
                 }
```
Comment (soninaren) [inline Bindings/bindings.json]: selectConnectoin typo at all the place holders

Comment (alrod) [inline Bindings/bindings.json]: fixed with e0d3a7c8d11329ef99ff6d187c603d4453c54d4f

File: Bindings/bindings.json [hunk 3]
```diff
@@ -50,7 +51,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Event Hub connection",
-                    "help": "This is the connection string for your Event Hub. Make sure that it has Receive permissions."
+                    "help": "This is the connection string for your Event Hub. Make sure that it has Receive permissions.",
```
Comment (soninaren) [inline Bindings/bindings.json]: This connection string in for service bus, ,may be the text "This is the Service Bus connections string" might be more suitable.

File: Bindings/bindings.json [hunk 4]
```diff
@@ -302,7 +310,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Service Bus connection",
-                    "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights."
+                    "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights.",
```
Comment (soninaren) [inline Bindings/bindings.json]: should be "Service Bus" instead of "Service Bus account"

File: Bindings/bindings.json [hunk 5]
```diff
@@ -522,7 +535,8 @@
                     "resource": "DocumentDB",
                     "required": true,
                     "label": "DocumentDB account connection",
-                    "help": "This is the connection string for your DocumentDB account."
+                    "help": "This is the connection string for your DocumentDB account.",
```
Comment (soninaren) [inline Bindings/bindings.json]: removed "account" from the string.

**Comment by soninaren** [comment]
@alrod i have added some comments.
@mattchenderson  would be the right person to review the strings in detail.

File: Bindings/bindings.json [hunk 6]
```diff
@@ -3,7 +3,8 @@
     "contentVersion": "2016-03-04-alpha",
     "variables": {
         "storageConnStringLabel": "Storage account connection",
-        "appSettingsHelp": "This is the connection string for your storage account."
+        "appSettingsHelp": "This is the connection string for your storage account.",
+        "selectConnection":  "Click select to choose a connection"
     },
```
Comment (soninaren) [inline Bindings/bindings.json]: nit: Add "." period at the end of the string.

File: Bindings/bindings.json (hunk 4 above)
Comment (mattchenderson) [inline Bindings/bindings.json]: In that case, it should be all lowercase. "service bus" would be the correct form

File: Bindings/bindings.json [hunk 7]
```diff
@@ -333,7 +342,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Service Bus connection",
-                    "help": "This is the connection string for your Service Bus account. Make sure that it has Send rights."
+                    "help": "This is the connection string for your Service Bus account. Make sure that it has Send rights.",
```
Comment (mattchenderson) [inline Bindings/bindings.json]: Same as above. "Service Bus account" -> "service bus"

File: Bindings/bindings.json (hunk 5 above)
Comment (mattchenderson) [inline Bindings/bindings.json]: Actually, this should stay. DocumentDB account is an actual entity.

File: Bindings/bindings.json (hunk 7 above)
Comment (mathewc) [inline Bindings/bindings.json]: Why are you guys lower casing these service names? It should be ServiceBus, just like we use DocumentDb, EventHub, etc.

File: Bindings/bindings.json [hunk 8]
```diff
@@ -481,7 +493,8 @@
                     "resource": "DocumentDB",
                     "required": true,
                     "label": "DocumentDB account connection",
-                    "help": "This is the connection string for your DocumentDb account."
+                    "help": "This is the connection string for your DocumentDb account.",
```
Comment (mathewc) [inline Bindings/bindings.json]: make casing consistent. Here you have Db, below you use DB (correctly)

File: Bindings/bindings.json [hunk 9]
```diff
@@ -619,7 +633,8 @@
                     "resource": "ServiceBus",
                     "required": true,
                     "label": "Notification hub namespace connection",
```
Comment (mathewc) [inline Bindings/bindings.json]: Make casing consistent. Should be Notification Hub here and below. Actually the service name is "Notification Hubs" plural. We should probably use that, no?

File: Bindings/bindings.json (hunk 4 above)
Comment (mathewc) [inline Bindings/bindings.json]: Why would we ever use lower case for Service Bus?

File: Bindings/bindings.json (hunk 3 above)
Comment (mattchenderson) [inline Bindings/bindings.json]: "Event Hub" -> "event hub"

File: Bindings/bindings.json (hunk 7 above)
Comment (mattchenderson) [inline Bindings/bindings.json]: Instance names are always lowercase per Azure style guidelines. Only capitalized when referring to the brand.

File: Bindings/bindings.json (hunk 9 above)
Comment (mattchenderson) [inline Bindings/bindings.json]: Correct. In this case it should be "Notification Hubs namespace"

File: Bindings/bindings.json (hunk 4 above)
Comment (mattchenderson) [inline Bindings/bindings.json]: Instance names are always lowercase per Azure style guidelines. Only capitalized when referring to the brand.

**Pull request merged**